### PR TITLE
v1beta1 support for clustertask start command

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds v1beta1 support for clustertask start command
with other refactor for test

Signed-off-by: Pradeep Kumar pradkuma@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/cmd/clustertask/clustertask_test.go [hunk 1]
```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/tektoncd/cli/pkg/test"
 )
 
+const (
```
Comment (vdemeester) [inline pkg/cmd/clustertask/clustertask_test.go:23]: Is it used ? :thinking:

File: pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_run_with_no_output.golden [hunk 2]
```diff
@@ -6,13 +6,12 @@ metadata:
   labels:
     key: value
 spec:
-  inputs:
-    resources:
+  resources:
```
Comment (piyush-garg) [inline pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_run_with_no_output.golden:9]: this is wrong spec for v1alpha1, need to also handle the dryrun like create

File: pkg/cmd/clustertask/clustertask_test.go (hunk 1 above)
Comment (pradeepitm12) [reply pkg/cmd/clustertask/clustertask_test.go:23]: Yes, it is used in tests.

File: pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_run_with_no_output.golden (hunk 2 above)
Comment (pradeepitm12) [reply pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_run_with_no_output.golden:9]: Thanks for this one. I will update the PR.
We will need to handle this in task start as well.

**Comment by pradeepitm12** [comment]
/hold 
until dry run corrected

**Pull request merged**